packages/aragon-wrapper/src/rpc/handlers/index.js

```diff
@@ -1,6 +1,6 @@
 import { Observable } from 'rxjs/Rx'
 
-export function createResponse ({ request: { id } }, { error, value = null }) {
```

Comment:
I removed the default assignment here from `null` because it was causing duplicates to be emitted.
(As long as they remain `undefined` they will get filtered in `createRequestHandler`)

Reply:
I think the reason they're `null` is because JSON-RPC can't transmit `undefined` values over the wire.

Reply:
Oops, nice catch!
I fixed the duplicates by explicitly checking `kind === 'C'`.
Also added a test check for `undefined` results that should be converted to `null`.